apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3766,17 +3766,23 @@ static Observation findOrCreateContainedObservation(ExplanationOfBenefit eob, St
    * @param id The resource ID
    * @return The found or new {@link Organization} resource
    */
-  static Resource findOrCreateContainedOrg(ExplanationOfBenefit eob, String id) {
-    Optional<Resource> org = eob.getContained().stream().filter(r -> r.getId() == id).findFirst();
+  static Organization findOrCreateContainedOrganization(ExplanationOfBenefit eob, String id) {
+    Optional<Resource> organization =
+        eob.getContained().stream().filter(r -> r.getId() == id).findFirst();
```

Comment:
Are these non unit test related changes meant to be in this PR?

Reply:
yes

Reply:
i wanted to refactor some of the functions that gets hit.. Just made more sense to me